python/llm/example/CPU/Speculative-Decoding/baichuan2/README.md

```diff
@@ -60,3 +60,16 @@ Tokens generated 128
 E2E Generation time x.xxxxs
 First token latency x.xxxxs
 ```
+
+### 4. (Optional) Enable BIGDL_OPT_IPEX
```

Comment:
Optional? Or add this part into the former section, and thus require users to apply the optimization immediately.